flang/test/Lower/percent-val-actual-argument.f90

```diff
@@ -0,0 +1,14 @@
+! RUN: bbc %s -emit-fir -o - | FileCheck %s
```

Comment:
```suggestion
! RUN: bbc %s -emit-hlfir -o - | FileCheck %s
```

This doesn't make much of a difference here, but it is better to test the lowering code on its own and not lowering plus all of the passes to get from HLFIR to FIR.

These days most tests are written using `flang -fc1 -emit-hlfir` but that is more of a convention. I don't think it is wrong to use `bbc`.